# 문제4) 다음 코드는 오류가 발생하는지 설명하시오
num = 3


def printNum3():
    print(num)
    num = 5
    print(num)


printNum3()
